indico/web/forms/widgets.py

```diff
@@ -288,3 +288,13 @@ def __init__(self, show_field=True):
 
     def __call__(self, field, **kwargs):
         return super().__call__(field, input_args=kwargs)
+
+
+class TextCompleteWidget(JinjaWidget):
```

Comment:
What about `LabelledTextWidget`? `TextCompleteWidget` sounds like something with autocomplete to me...

Reply:
> Perhaps I could name it `PrefixedTextWidget`?

sounds fine as well

> Or even make it more generic and allow any field to be prefixed, instead of only an input?

i would not spend time on that. wtforms is semi-legacy, and there's a very good chance any fields that are not basic text input fields would not just work with something generic